src/main/java/mate/jdbc/util/Injector.java

```diff
@@ -10,6 +10,9 @@
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.lib.Inject;
+import mate.jdbc.lib.Service;
```

Comment:
return the original version of this class, you shouldn't modify it according to the task's description

Reply:
mvn clean paxkage does not allow me to pass this location of imports